This fixes a Python 3 doctest failure in
rings/polynomial/polynomial_integer_dense_flint.pyx: see #28334.
See https://github.com/wbhart/flint2/pull/585 for upstream PR

diff --git a/memory_manager.c b/memory_manager.c
index 04abf8a..bece9f5 100644
--- a/memory_manager.c
+++ b/memory_manager.c
@@ -41,6 +41,7 @@ pthread_mutex_t register_lock;
 static void flint_memory_error()
 {
     flint_printf("Exception (FLINT memory_manager). Unable to allocate memory.\n");
+    fflush(stdout);
     abort();
 }

